src/com/app/ActiveAccount.java

```diff
@@ -88,7 +84,7 @@ void setAccountsFromFile() {
         try {
             FileReader fileReader = new FileReader(file);
             String line;
-            this.accounts = new ArrayList<>();
+            Map<String, Account> accountMap = new HashMap<>();
 
             BufferedReader reader = new BufferedReader(fileReader);
             while ((line = reader.readLine()) != null) {
```

Comment:
Change the reading style to Java8 (you can find an example here https://mkyong.com/java8/java-8-stream-read-a-file-line-by-line/ )